internal/audit/kafka/tls.go

```diff
@@ -31,7 +31,10 @@ func NewTLSConfig(ctx context.Context, reloadInterval time.Duration, insecureSki
 		return nil, fmt.Errorf("failed to read CA certificate: %w", err)
 	}
 
-	caCertPool := x509.NewCertPool()
+	caCertPool, err := x509.SystemCertPool()
+	if err != nil {
+		caCertPool = x509.NewCertPool()
+	}
 	if !caCertPool.AppendCertsFromPEM(caCert) {
```

Comment:
`caPath` should likely be made optional as well, so it defaults to loading systems certs